frontend/src/components/Pages/FrontPage/AutoScheduleSection/AutoScheduleMissionTableRow.tsx

```diff
@@ -15,10 +15,36 @@ const StyledTableRow = styled.div`
     grid-template-columns: 100px auto 100px;
 `
 
-const skipAutoScheduledMission = async (missionId: string, timeOfDay: string) => {
+export const skipAutoScheduledMission = async (missionId: string, timeOfDay: string) => {
     await BackendAPICaller.skipAutoScheduledMission(missionId, timeOfDay)
 }
 
+enum MissionStatusType {
+    ScheduledJob = 'ScheduledJob',
+    SkippedJob = 'SkippedJob',
+    PastJob = 'PastJob',
+    FutureUnstartedJob = 'FutureUnstartedJob',
+}
+
+const currentDayOfTheWeek = allDays[allDaysIndexOfToday]
+
+export const selectMissionStatusType = (day: DaysOfWeek, time: string, mission: MissionDefinition) => {
+    if (day !== currentDayOfTheWeek) {
+        return MissionStatusType.FutureUnstartedJob
+    }
+    if (time < convertUTCDateToLocalDate(new Date()).toISOString().substring(11, 19)) {
```

Comment:
```suggestion
    if (time < getNowAsTimeOnly() {
```
Would create a function that gives you the current time in the right format and use that here instead. Increases readability of the if-check.

Reply:
This is old code

Reply:
I have added it to the PR